### PR TITLE
website/docs: added content to Welcome page

### PR DESCRIPTION
**Pull request by tanberry**
# Details
Added more content about authentik's features/functionalities, and the interfaces.

### CODE REVIEW

File: website/docs/index.md [hunk 1]
```diff
@@ -5,14 +5,28 @@ slug: /
...
 ## Installation
 
-See [Docker-compose](installation/docker-compose) or [Kubernetes](installation/kubernetes)
+Refer to the installatin steps in either [Docker-compose](installation/docker-compose) or [Kubernetes](installation/kubernetes).
+
+For more information about configuration, Beta versions, and additional installation options, see our main [Installation](installation) section.
 
 ## Screenshots
 
+authentik can use Light or Dark mode for the Admin interface and the User interface.
```
Comment (BeryJu) [inline website/docs/index.md:28]: The light/dark theme is supported in all 3 interfaces, even though there's only screenshots for two of them

File: website/docs/index.md [hunk 2]
```diff
@@ -5,14 +5,28 @@ slug: /
...
+
+The authentik product provides the following consoles:
+
+* Admin Interface: a visual tool for the creation and management of users and groups, tokens and credentials, application integrations, events, and the Flows that define standard and customizable login and authentication processes. Easy-to-read visual dashboards display system status, recent logins and authenticatin events, and application usage.
+
+* User Interface: this console view in authentik displays all of the applications and integrations in which you have implemented authentik. Click on the app that you want to work on, and drill down into the specifics for that app.
+
+* Flow panels: [_Flows_](./flow) are the steps by which the various _Stages_ of a login and authetnication process occurs.  A stage represents a single verification or logic step in the sign-on process. authentik allows for the customization and exact definition of these flows.
```
Comment (BeryJu) [inline website/docs/index.md:18]:
```suggestion
* Flows: [_Flows_](./flow) are the steps by which the various _Stages_ of a login and authetnication process occurs.  A stage represents a single verification or logic step in the sign-on process. authentik allows for the customization and exact definition of these flows.
```

File: website/docs/index.md [hunk 3]
```diff
@@ -5,14 +5,28 @@ slug: /
...
+
+You can use authentik in an existing environment to add support for new protocols, so introducing authentik to your current tech stack doesn't present re-architecting challenges. We already support all of the major providers, such as OAuth2, SAML, LDAP, and SCIM, so you can pick the protocol that you need for each application.
+
+The authentik product provides the following consoles:
+
+* Admin Interface: a visual tool for the creation and management of users and groups, tokens and credentials, application integrations, events, and the Flows that define standard and customizable login and authentication processes. Easy-to-read visual dashboards display system status, recent logins and authenticatin events, and application usage.
+
+* User Interface: this console view in authentik displays all of the applications and integrations in which you have implemented authentik. Click on the app that you want to work on, and drill down into the specifics for that app.
```
Comment (BeryJu) [inline website/docs/index.md:16]:
```suggestion
* User Interface: this console view in authentik displays all of the applications and integrations in which you have implemented authentik. Click on the app that you want to access to open it, or drill down to edit its configuration in the admin interface
```

File: website/docs/index.md (hunk 1 above)
Comment (tanberry) [reply website/docs/index.md:28]: Yeah, I was wondering! I have a .png of the Welcome flow, and would like to addd it, but wanted to get your thought on that first?

Comment (tanberry) [reply website/docs/index.md:28]: Also I can do it in a subsequent PR, no rush now. :-)

Comment (BeryJu) [reply website/docs/index.md:28]: Yeah sure, feel free to add that here (make sure your browser is the same size so it matches the dimensions of the other screenshots)

Comment (tanberry) [reply website/docs/index.md:28]: Precisely because I am rather dreadful with graphics and sizings and resolutions, I'll do it in separate PR.

Comment (BeryJu) [reply website/docs/index.md:28]: Actually this reminds me of some other change I wanted to do for this, I'll PR those images too

Comment (BeryJu) [reply website/docs/index.md:28]: https://github.com/goauthentik/authentik/pull/4997

File: website/docs/index.md [hunk 4]
```diff
@@ -5,14 +5,28 @@ slug: /
...
+
+The authentik product provides the following consoles:
+
+* Admin Interface: a visual tool for the creation and management of users and groups, tokens and credentials, application integrations, events, and the Flows that define standard and customizable login and authentication processes. Easy-to-read visual dashboards display system status, recent logins and authentication events, and application usage.
+
+* User Interface: this console view in authentik displays all of the applications and integrations in which you have implemented authentik. Click on the app that you want to access to open it, or drill down to edit its configuration in the admin interface
+
+* Flow panels: [_Flows_](./flow) are the steps by which the various _Stages_ of a login and authentication process occurs.  A stage represents a single verification or logic step in the sign-on process. authentik allows for the customization and exact definition of these flows.
```
Comment (tanberry) [inline website/docs/index.md:18]:
```suggestion
* Flows: [_Flows_](./flow) are the steps by which the various _Stages_ of a login and authentication process occurs.  A stage represents a single verification or logic step in the sign-on process. authentik allows for the customization and exact definition of these flows.
```

File: website/docs/index.md [hunk 5]
```diff
@@ -5,14 +5,28 @@ slug: /
...
+
+You can use authentik in an existing environment to add support for new protocols, so introducing authentik to your current tech stack doesn't present re-architecting challenges. We already support all of the major providers, such as OAuth2, SAML, LDAP, and SCIM, so you can pick the protocol that you need for each application.
+
+The authentik product provides the following consoles:
+
+* Admin Interface: a visual tool for the creation and management of users and groups, tokens and credentials, application integrations, events, and the Flows that define standard and customizable login and authentication processes. Easy-to-read visual dashboards display system status, recent logins and authentication events, and application usage.
```
Comment (tanberry) [inline website/docs/index.md:14]:
```suggestion
* **Admin interface**: a visual tool for the creation and management of users and groups, tokens and credentials, application integrations, events, and the Flows that define standard and customizable login and authentication processes. Easy-to-read visual dashboards display system status, recent logins and authentication events, and application usage.
```

File: website/docs/index.md [hunk 6]
```diff
@@ -5,14 +5,28 @@ slug: /
...
+
+You can use authentik in an existing environment to add support for new protocols, so introducing authentik to your current tech stack doesn't present re-architecting challenges. We already support all of the major providers, such as OAuth2, SAML, LDAP, and SCIM, so you can pick the protocol that you need for each application.
+
+The authentik product provides the following consoles:
+
+* **Admin interface**: a visual tool for the creation and management of users and groups, tokens and credentials, application integrations, events, and the Flows that define standard and customizable login and authentication processes. Easy-to-read visual dashboards display system status, recent logins and authentication events, and application usage.
+
+* User Interface: this console view in authentik displays all of the applications and integrations in which you have implemented authentik. Click on the app that you want to access to open it, or drill down to edit its configuration in the admin interface
```
Comment (tanberry) [inline website/docs/index.md:16]:
```suggestion
* **User interface**: this console view in authentik displays all of the applications and integrations in which you have implemented authentik. Click on the app that you want to access to open it, or drill down to edit its configuration in the admin interface
```

File: website/docs/index.md [hunk 7]
```diff
@@ -5,14 +5,28 @@ slug: /
...
+
+The authentik product provides the following consoles:
+
+* **Admin interface**: a visual tool for the creation and management of users and groups, tokens and credentials, application integrations, events, and the Flows that define standard and customizable login and authentication processes. Easy-to-read visual dashboards display system status, recent logins and authentication events, and application usage.
+
+* **User interface**: this console view in authentik displays all of the applications and integrations in which you have implemented authentik. Click on the app that you want to access to open it, or drill down to edit its configuration in the admin interface
+
+* Flows: [_Flows_](./flow) are the steps by which the various _Stages_ of a login and authentication process occurs.  A stage represents a single verification or logic step in the sign-on process. authentik allows for the customization and exact definition of these flows.
```
Comment (tanberry) [inline website/docs/index.md:18]:
```suggestion
* **Flows**: [_Flows_](./flow) are the steps by which the various _Stages_ of a login and authentication process occurs.  A stage represents a single verification or logic step in the sign-on process. authentik allows for the customization and exact definition of these flows.
```

File: website/docs/index.md (hunk 1 above)
Comment (tanberry) [reply website/docs/index.md:28]: ohhhh nice images. But... why not add to the PR, to avoid a merge conflict?

**Review by BeryJu — APPROVED** [review]

**Pull request merged**